src/main/scala/is/hail/variant/ReferenceGenome.scala

```diff
@@ -527,8 +527,9 @@ case class ReferenceGenome(name: String, contigs: Array[String], lengths: Map[St
 
 object ReferenceGenome {
   var references: Map[String, ReferenceGenome] = Map()
-  val GRCh37: ReferenceGenome = fromResource("reference/grch37.json")
-  val GRCh38: ReferenceGenome = fromResource("reference/grch38.json")
+  val GRCh37: ReferenceGenome = fromResource("reference/GRCh37.json")
```

Comment:
these should still be all lowercase names, right? Or did you also mean to change those file names?

Reply:
I did, I think they should be consistent with the reference genome names. can revert if you want

Reply:
no, that looks good---I just don't see the renames in this PR.

Reply:
huh, looks like my file moves never got checked in. odd.

Reply:
will just revert